docs/Course Wiki/PHYS Course Offerings/PHYS214.md

```diff
@@ -39,14 +39,18 @@ In the past, this course has been taught by Professors Wagner and Covey.
 
 ## Course Tips
 
-The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study.
+The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study. Also, the discussion worksheets are the most difficult, then the homework, then the exams. So, if you understand the concepts in the discussion worksheets and homework, you will be fine in the exam.
```

Comment:
We should probably rewrite this as "The ranking of difficulty of problems is: Discussion Worksheets, Homework Problems, Exams."